blockchain/source/migrations/4_init_gatekeeper.js

```diff
@@ -0,0 +1,23 @@
+var SimpleGatekeeper = artifacts.require('./SimpleGatekeeper.sol');
+var SNM = artifacts.require('./SNM.sol');
+
+module.exports = function (deployer, network) {
+    if (network === 'private') {
+        SNM.deployed()
+            .then(function (res) {
+                res.transfer(SimpleGatekeeper.address, 444 * 1e6 * 1e18, { gasPrice: 0 });
+            })
+            .catch(function (err) {
+                console.log(err);
+            });
+    } else if (network === 'rinkeby') {
+    } else {
```

Comment:
wow, such `else if`, so empty

Reply:
it's ok. Deployer must support this section.